contracts/lib/Structs.sol

```diff
@@ -58,10 +59,11 @@ library Structs {
         bool active;
         uint8 id;
         uint8 assetType;
+        int8 power;
+        uint32 epoch;
+        uint32 aggregationMethod;
         uint8[] jobIDs;
```

Comment:
same

Reply:
hmm here keeping JobIDs later is better I think as per the reason mentioned above
let me know what you think, will make a change accordingly